lib/web_ui/lib/src/engine/semantics/semantics.dart

```diff
@@ -849,30 +849,14 @@ class SemanticsObject {
     final bool hasIdentityTransform =
         transform == null || isIdentityFloat32ListTransform(transform);
 
+    // On Mac OS and iOS, VoiceOver requires left=0 top=0 value to correctly
+    // handle order. See https://github.com/flutter/flutter/issues/73347.
```

Comment:
Should this comment be closer to the `_resetContainerOffsets` and `_resetElementOffsets` functions?

Reply:
done